docs/content/intro/community-roles.md

```diff
@@ -0,0 +1,105 @@
+
+Read on to find out more!
+
+# Community Members
+
+Community members are folks who decide they want to get involved with our
+community. Absolutely anyone can do that whenever they want. If you want
+to get involved, that doesn't mean you have to continue being involved, but
+we hope our community is welcoming and the work is interesting enough to 
+convince you to say :)
+
+We'll provide all the support we can possibly provide to help you contribute
+in any way you'd like. If you're considering joining us, here are some ideas
+for how you can get involved:
+
+- Comment on an [issue](https://github.com/gomods/athens/issues) that you're
+interested in
+- Submit a [pull request](https://github.com/gomods/athens/pulls) (PR) to fix 
+an issue, or to improve something that doesn't have an issue
+- Review a PR that you're interested in
+- Join us at a weekly [development meeting](https://docs.google.com/document/d/1xpvgmR1Fq4iy1j975Tb4H_XjeXUQUOAvn0FximUzvIk/edit#)
+(or more than one!)
+- Come chat with us in the [gophers slack](https://invite.slack.golangbridge.org/) in the `#athens` channel
+- ... and anything else that's appropriate for you!
+
+# Contributors
+
+As you participate in the community more and more, you'll have the opportunity 
+to become a contributor. Here's what being a contributor means, and what you
+should do to become one.
+
+## What Being a Contributor Means
+
+As a contributor, you'll have issues assigned to you and you'll be requested 
+to review pull requests (PRs) via the 
+[Github pull request review system](https://help.github.com/articles/about-pull-request-reviews/). 
+We rely heavily on Github PR reviews, so if you review a PR as a contributor, you can decide 
+when that PR is ready to be merged.
+
+## How to Become a Contributor
+
+To become a contributor, the core maintainers of the project would like to see you:
+
+- Attend our development meetings regularly<sup>1</sup>
+- Comment on issues with your experiences and opinions
+- Add your comments and reviews on pull requests (anyone can do this as a community member)
+- Contribute PRs to fix issues
+- Open issues as you find them
+
+# Maintainers
+
+After you become a contributor, you'll have the opportunity to become a maintainer.
+Here's what being a maintainer means and how to become one.
+
+## What Being a Maintainer Means
+
+As a maintainer, you'll be doing the same things as a contributor with a few 
+extras:
+
+- Help organize our development meetings (i.e. help organize the agenda)
+- Promote the project and build community (e.g. present on it where possible, write about it, ...) when possible<sup>2</sup>
+- Triage issues (e.g. adding labels, promoting discussions, finalizing decisions)
+- Organize and promote PR reviews (e.g. prompting community members, contributors, and other maintainers to review)
+- Handle code of conduct issues
+
+## How to Become a Maintainer
+
+To become a maintainer, we would like you to see you be an effective contributor,
```

Comment:
Same here, it would help if we gave a hint on how to go about asking, and when it would be OK to ask, or if they should wait to be noticed, etc.

Reply:
What do folks think about the regular meeting + nominations like I mentioned in https://github.com/gomods/athens/pull/444/files#r211004524?

Reply:
I think it's good, and with the new private channel we can just mention if we notice someone as well.

Reply:
:+1: